changelog.json

```diff
@@ -1,5 +1,53 @@
+      "[Fixed] Icon accessibility labels fail when multiple icons are visible at the same time - #7174",
+      "[Fixed] Incorrectly encoding URLs affects issue filtering - #7506",
+      "[Fixed] License templates do not end with newline character - #6999",
+      "[Fixed] Merge and rebase conflicts banners do not hide after aborting operation outside Desktop - #7046",
+      "[Fixed] Missing tooltips for change indicators in the sidebar - #7174",
+      "[Fixed] Mistaken classification of all crashes being related to launch - #7126",
+      "[Fixed] Unable to switch keyboard layout and retain keyboard focus while using commit form - #6366. Thanks @AndreiMaga!",
+      "[Fixed] Prevent console errors due to underlying component unmounts - #6970",
+      "[Fixed] Menus disabled by activity in inactive repositories - #6313",
+      "[Fixed] Race condition with Git remote lookup may cause push to incorrect remote - #6986",
+      "[Fixed] Restore GitHub Desktop to main screen if external monitor removed - #7418 #2107. Thanks @say25!",
+      "[Fixed] Tab Bar focus ring outlines clip into other elements - #5802. Thanks @Daniel-McCarthy!",
+      "[Improved] \"Automatically Switch Theme\" on macOS does not check theme on launch - #7116. Thanks @say25!",
+      "[Improved] \"Add\" button in repository list should always be visible - #6646",
+      "[Improved] Pull Requests list loads and updates pull requests from GitHub more quickly - #7501 #7163",
+      "[Improved] Indicator hidden in Pull Requests list when there are no open pull requests - #7258",
+      "[Improved] Manually refresh pull requests instead of having to wait for a fetch - #7027",
+      "[Improved] Accessibility attributes for dialog - #6496. Thanks @HirdayGupta!",
+      "[Improved] Alignment of icons in repository list - #7133",
+      "[Improved] Command line interface warning when using \"github open\" with a remote URL - #7452. Thanks @msztech!",
+      "[Improved] Error message when unable to publish private repository to an organization - #7472",
+      "[Improved] Initiate cloning by pressing \"Enter\" when a repository is selected - #6570. Thanks @Daniel-McCarthy!",
+      "[Improved] Lowercase pronoun in \"Revert this commit\" menu item - #7534",
+      "[Improved] Styles for manual resolution button in \"Resolve Conflicts\" dialog - #7302",
+      "[Improved] Onboarding language for blank slate components - #6638. Thanks @jamesgeorge007!",
+      "[Improved] Explanation for manually conflicted text files in diff viewer - #7611",
+      "[Improved] Visual progress on \"Remove Repository\" and \"Discard Changes\" dialogs - #7015. Thanks @HashimotoYT!",
+      "[Improved] Menu items now aware of force push state and preference to confirm repository removal - #4976 #7138",
+      "[Removed] Branch and pull request filter text persistence - #7437",
+      "[Removed] \"Discard all changes\" context menu item from Changes list - #7394. Thanks @ahuth!"
```

Comment:
If we get #7650, this changes from `Removed` to `Improved` right? Just a thought.

Reply:
This branch was cut before #7650, but I split this into two items:

 - adding the new menu item 

https://github.com/desktop/desktop/blob/cef2fe7b21e51a178c3da626b349cfe21303609c/changelog.json#L10

 - removing the context menu item in Changes (this entry)

A follow-up release containing #7650 will then mention the keyboard shortcut. That seems the best way tot follow along with the changes in this area...